cli/tests/pcluster/delete/test_pcluster_delete.py

```diff
@@ -0,0 +1,105 @@
+"""This module provides unit tests for the functions in the pcluster.utils module."""
```

Comment:
The comment seems pasted from the test_pcluster_utils.py

Reply:
Done.